assets/global.scss

```diff
@@ -14,7 +14,7 @@
 }
 
 *::-webkit-scrollbar-thumb {
-  background-color: var(--v-thumb-base);
```

Comment:
Isn't the --webkit decorator added automatically at build time?

Reply:
If it's not, we should fix that as soon as possible.

Reply:
This is a special vendor prefix that is not on a standard track, so it's not handled by autoprefixer.